api/routes/index.js

```diff
@@ -1,4 +1,27 @@
-import express from 'express';
-const router = express.Router();
+export function register(server, options, next) {
+  /* Serve static files from public folder */
+  server.route({
```

Comment:
Kind of like how we have done it here. https://github.com/RedCometLabs/hapi-react-couch/blob/master/server.js#L44